packages/medusa/src/services/order.ts

```diff
@@ -663,25 +664,23 @@ class OrderService extends TransactionBaseService {
       let gcBalance =
```

Comment:
suggestion: Maybe we can remove it to something like `giftCardableAmount`. It looks more like the amount that can be gifted than the balance itself. wdyt?